packages/lit-html/src/lit-html.ts

```diff
@@ -262,11 +266,14 @@ export const render = (
   container: HTMLElement | DocumentFragment,
   options?: RenderOptions
 ) => {
-  let part: NodePart = (container as any).$lit$;
+  let part: NodePart = (options?.renderBefore ?? (container as any)).$lit$;
   if (part === undefined) {
-    const marker = createMarker();
-    container.append(marker);
-    (container as any).$lit$ = part = new NodePart(marker, null, options);
+    const end = options?.renderBefore ?? null;
+    (container as any).$lit$ = part = new NodePart(
```

Comment:
Doesn't this need to change too?

I think we want something like:

```ts
const partOwnerNode = options?.renderBefore ?? container;
let part = (partOwnerNode as any).$lit$;
if (part === undefined) {
  const endNode = options?.renderBefore ?? null;
  (partOwnerNode as any).$lit$ = ...
}
```

Reply:
Oops, thanks.